templates/birds/macros.j2

```diff
@@ -6,36 +6,40 @@
   {% set author = page.config.bird %}
 
   <div data-module='bird' class="h-card" id="bird-{{ author }}">
-    {{ utility.icon(author, size='face') }}
+    <div class="bird-face">
+      {{ utility.icon(author, size='face') }}
+    </div>
 
     <header>
       <h2 class="h-card">
         {{ birds.name_link(author) }}
       </h2>
 
-      <div>
+      <ul class="socials">
         {% set links = page.config.social[0] %}
         {% for link in links %}
-          {{ birds.get_social(link, links[link]) }}
+          <li>{{ birds.get_social(link, links[link]) }}</li>
         {% endfor %}
-      </div>
+      </ul>
     </header>
 
-    {{ page.render_contents() }}
+    <div class="bio">
+      {{ page.render_contents() }}
+    </div>
 
     {{ oss_list(author, oss) }}
     {{ blog_list(
       posts=get_blog_entries_by_bird(author)|reverse|list
-      ) }}
+    ) }}
   </div>
 {% endmacro %}
 
 
 {% macro blog_list(posts) %}
   {% if posts|length %}
-    <div>
+    <div class="bird-posts">
       <h3>Recent Articles</h3>
-      <ul>
+      <ul class="title-list">
```

Comment:
We aren't using this class anywhere yet, but I feel like we will. Same with bird-posts. Also, the h3 should probably have a class too, but I'm not sold yet on naming it.